src/uct/api/uct.h

```diff
@@ -1318,6 +1313,16 @@ ucs_status_t uct_ep_connect_to_ep(uct_ep_h ep, const uct_device_addr_t *dev_addr
  *                               remote peer.
  * @param [in]  length           Length of the private data.
  * @param [out] ep_p             Handle to the created endpoint.
+ *
+ * @return UCS_OK              - The connection to the server, on the client side,
+ *                               was established. No reply from the server is
+ *                               required and therefore the reply_cb callback
+ *                               won't be invoked.
+ *         UCS_INPROGRESS      - The connection to the remote peer was initiated.
+ *                               The user will be notified of the connection
+ *                               establishment, on the client side, when the reply_cb
+ *                               callback will be invoked.
+ *         error code          - In case of an error.
```

Comment:
Suggest modifying to:
  \return \ref ucs_status_t "Error code" - In case of issues.